pallets/staking/src/lib.rs

```diff
@@ -1497,7 +1505,13 @@ decl_error! {
         /// Updates with same value.
         NoChange,
         /// Updates with same value.
-        InvalidCommission
+        InvalidCommission,
+        /// Given potential entity has invalid identity.
```

Comment:
Not sure what this means; can you rephrase?

Reply:
I will fix this in Commission PR #670